### DISCUSSION TITLE
Build in Pypi are too old.

### DISCUSSION THREAD

**Discussion by jamestjsp** [ilayn/harold#72] [Q&A] [answered]
Hello Ilayn, 
Is there any plans to release a new version of  harold in Pypi?
Kind regards
James

**Comment by ilayn** [comment, marked as answer]
I would love to! Let me spill out the situation properly maybe for others too. 


I have two major blocks. One of them is the annoying root locus functionality. And the other one is #68 which turned out to be annoying. Internally I need to make sure that the impulse response, step response etc. accepts multiple systems just like bode does now on the master branch. That would definitely make things much more robust. 

In the meantime, I think everybody is using python-control anyways (which is perfectly fine) and some of harold code went in there too and I am also a maintainer of SciPy. So I am not so sure about the worth of what harold brings to the table. 

But you are right, at least, on PyPi it should be a robust and something closer to the master branch version.

**Reply by jamestjsp** [reply]
The problem with control package is the dependency on slycot for MIMO system.

**Reply by ilayn** [reply]
That's actually one of the reasons why I started this library. Because for the life of me, I can't install it on Windows and it is GPL licensed so I can't peek in. But recently a miracle happened and https://github.com/SLICOT/SLICOT-Reference is now BSD3 licensed.

**Reply by jamestjsp** [reply]
That's a problem to write a production code. "A permissive license similar to the BSD 2-Clause License, but with a 3rd clause that prohibits others from using the name of the project or its contributors to promote derived products without written consent."

**Reply by ilayn** [reply]
If you include the copyright statements, there is no problem with BSD3 license in a commercial setting. Many companies use excessive amount of BSD3 licensed code including scipy in their production code. What you quote is that companies cannot promote "we are using SciPy in our tech" without consent.
